query-engine/connector-test-kit/src/test/scala/queries/nativeTypes/nativeTypesWithPSLFeaturesOnPostgres.scala

```diff
@@ -0,0 +1,445 @@
+        }
+        assert(database.setupWithStatusCode(project) == 0)
+      }
+  }
+
+  "Using Prisma scalar type Bytes with native types and PSL features" should "work" in {
+    val prisma_type = Vector("Bytes")
+    val native_type = Vector("ByteA")
+    for (p_type <- prisma_type;
+         n_type <- native_type
+         )
+      yield {
+        val project = SchemaDsl.fromStringV11() {
+          s"""
+             |model User2 {
+             |  email    String  @unique
+             |  name     $p_type @test.$n_type
+             |  optional $p_type? @test.$n_type
+             |}
+             |
+             |model Item {
+             |  id   $p_type @test.$n_type @id
+             |  test $p_type @test.$n_type @unique
+             |  optional $p_type? @test.$n_type
+             |}
+             |
+             |model Post {
+             |  firstName $p_type @test.$n_type
+             |  lastName  $p_type @test.$n_type
+             |  email     $p_type @test.$n_type @unique
+             |  @@id([firstName, lastName])
+             |}
+             |
+             |model User {
+             |  id        Int     @default(autoincrement()) @id
+             |  firstname $p_type @test.$n_type
+             |  lastname $p_type @test.$n_type
+             |  @@unique([firstname, lastname])
+             |}
+             |
+             |model SingleColumnIndex {
+             |  id       Int     @id @default(autoincrement())
+             |  title    $p_type @test.$n_type
+             |  @@index([title])
+             |}
+             |
+             |model MultiColumnIndex {
+             |  id       Int     @id @default(autoincrement())
+             |  title    $p_type @test.$n_type
+             |  content  $p_type? @test.$n_type
+             |  @@index([title, content])
+             |}
+             |
+             |model House {
+             |  id $p_type @test.$n_type @id
+             |}
+    """.stripMargin
+        }
+        assert(database.setupWithStatusCode(project) == 0)
+      }
+  }
+
+  "Using Prisma scalar type datetime with native type and PSL features" should "work" in {
+    val prisma_type = Vector("DateTime")
+    val native_type = Vector( "Time(1)", "Timestamp(2)", "TimestampWithTimeZone(2)",  "Date","TimeWithTimeZone(4)")
+    for (p_type <- prisma_type;
+         n_type <- native_type
+         )
+      yield {
+        val project = SchemaDsl.fromStringV11() {
+          s"""
+             |model User {
+             |  id    Int  @id
+             |  optional $p_type? @test.$n_type
+             |}
+             |
+             |model Post {
+             |   id Int @id
+             |   time $p_type @test.$n_type @updatedAt
+             |}
+             |
+             |model Sun {
+             |  id Int @id
+             |  name     $p_type @test.$n_type
+             |}
+    """.stripMargin
+        }
+        assert(database.setupWithStatusCode(project) == 0)
+      }
+  }
+
```

Comment:
This is a Mysql error in the postgres file, maybe a copy artifact?

Reply:
Ah, yeah I copied the error message, but ` internal error: entered unreachable code: NOW default on non-datetime column"` is the same for both MySQL and Postgres